docs/install/openshift.md

```diff
@@ -0,0 +1,263 @@
+## Requirements
+
+Before proceeding, please ensure that you have an OpenShift cluster running K8s
+1.19+ (OpenShift 4.7+) and have Helm 3.5+ installed. In addition, you'll need to
+install the OpenShift CLI (`oc`) to authenticate to your cluster and create OpenShift
+resources.
+
+You'll also want to install the [latest version of Coder](https://github.com/coder/coder/releases/latest)
+locally in order to log in and manage templates.
+
+## Install Coder with OpenShift
+
+### 1. Authenticate to OpenShift and create a Coder project
+
+Run the following command to login to your OpenShift cluster:
+
+```console
+oc login <cluster-url>
+```
+
+This will configure your local `~/.kube/config` file with the cluster credentials
+needed when installing Coder via `helm`.
+
+Next, you will run the below command to create a project for Coder:
+
+```console
+oc new-project coder
+```
+
+### 2. Configure SecurityContext values
+
+Depending upon your configured Security Context Constraints (SCC), you'll need to set
+the following `securityContext` values in the Coder Helm chart:
+
+```yaml
+coder:
+  securityContext:
+    runAsNonRoot: true
+    runAsUser: <project-specific UID>
+    runAsGroup: <project-specific GID>
+    readOnlyRootFilesystem: false
+    seccompProfile:
+      type: RuntimeDefault
+    allowPrivilegeEscalation: false
+    capabilities:
+      drop:
+        - ALL
+```
+
+The above values are the Coder defaults. You will need to change these values in
+accordance with the applied SCC. To get a current list of SCCs, run the below command:
+
+```console
+oc get scc
+```
+
+> Note: you must have cluster-admin privileges to manage SCCs
+
+### 3. Set the `CODER_CACHE_DIRECTORY` environment variable
+
+By default, Coder creates the cache directory in `/home/coder/.cache`. Given the
+OpenShift-provided UID, the Coder container does not have permission to write to
+this directory.
+
+To address this issue, you will need to set the `CODER_CACHE_DIRECTORY` environment
+variable in your Helm values to the following:
+
+```yaml
+coder:
+  env:
+    - name: CODER_CACHE_DIRECTORY
+      value: /tmp/coder-cache
+```
+
+### 4. Set access URL, PostgreSQL connection values
+
+Set the `CODER_PG_CONNECTION_URL` value to enable Coder to establish a connection
+to a PostgreSQL instance. [See our Helm documentation](./kubernetes.md) on configuring
+the PostgreSQL connection URL as a secret. Additionally, if accessing Coder over a hostname, set
+the `CODER_ACCESS_URL` value.
```

Comment:
Isn't it more that say the CODER_ACCESS_URL should be set, like with any install? This comes across as optional, while technically true in some case.

Reply:
@sharkymark if not set in the values file, `CODER_ACCESS_URL` will automatically be set by Coder to the `cluster.local` DNS name, so we don't need to say it's a requirement, per se.